src/entry.ts

```diff
@@ -75,7 +80,9 @@ export interface StructuredJson {
   // Universally supported properties
   message?: string | object;
   httpRequest?: object;
-  timestamp?: string;
+  // Based on https://cloud.google.com/logging/docs/agent/logging/configuration#timestamp-processing, the
```

Comment:
Is there a reason you chose to use the seconds/nanos dictionary option instead of RFC 3339? It seems like it should be pretty simple to convert from a Date object to that format: https://codetogo.io/how-to-create-date-in-rfc3339-format-in-javascript/

Reply:
Please see the reference to spec I shared in description - [timestamp-processing](https://cloud.google.com/logging/docs/agent/logging/configuration#timestamp-processing).
Basically, RFC you mentioned could be placed only to `time` field - if we want to continue to use `timestamp` field, it must be nanos/seconds format. Let me know if you think we should use the `time` field with in format defined in RFC.

Reply:
I see. Note that on the backend the `time` field will become a `timestamp`, so it should be the same to the user:

> The Logging agent can receive and process time-related fields in several JSON formats.
>  If one of the following JSON timestamp representations is present in a structured record, 
> **the Logging agent strips the fields from jsonPayload and collapses them into a single 
> representation in the timestamp field in the LogEntry object**

Personally I think using the RFC format with a time field is easier, but I'm fine either way

Reply:
I wanted to keep `StructuredJson.timestamp` just to make sure I am making minimal change existing structure. Also worth mentioning that `EntryJson.timestamp` already written in nanos/seconds format and seems it works today (while `StructuredJson.timestamp` is not processed today due to wrong format).